server/storage/database.go

```diff
@@ -63,6 +57,32 @@ func (db *MySQLStorage) UpdateCurrent(gun, role string, version int, data []byte
 	return nil
 }
 
+// UpdateMany atomically updates many TUF records in a single transaction
+func (db *MySQLStorage) UpdateMany(gun string, updates []MetaUpdate) error {
+	insertStmt := "INSERT INTO `tuf_files` (`gun`, `role`, `version`, `data`) VALUES (?,?,?,?) WHERE (SELECT count(*) FROM `tuf_files` WHERE `gun`=? AND `role`=? AND `version`>=?) = 0;"
```

Comment:
Why do we need this `WHERE` clause? Shouldn't this just be a DB uniqueness constraint on (gun, role, version)?